### ISSUE TITLE
[BUG] API Method Path gets messed up when the ResourcePath in Swagger matches first part of method path,

### ISSUE THREAD

**Issue by stevebower59** [Axway-API-Management-Plus/apimanager-swagger-promote#285]
**Swagger-Promote version 1.6.7-1**
> Please make sure to use the latest Swagger-Promote version.

**API Manager version 7.6.2SP3**

**Expected behavior**

When we have a Resource Path like:
"resourcePath" : "/poreceipt",

and an API Method Path like: 

"path" : "/poreceipts",

The imported value of the API Method path shows up as **/poreceipt/s** 
where the expected Behaviour is to append ResourcePath to API Method Path 
i.e **/poreceipt/poreceipts**

**Actual behavior**
The API Method Path shows up as **/poreceipt/s** in the API Manager Catalog Backend.  The API method path should be **/poreceipt/poreceipts**.  It should append API Method Path to Resource Path.  

**Log-Output in Debug**

**Test case sample**

> Please describe the steps you have done to replicate the issue
Exported and imported with version 1.6.7-1. 

Tried with version 1.6.4, and replicated this issue with latest version 1.6.7-1

**Comment by cwiechmann** [comment]
Hi @stevebower59, thanks for reporting this issue. Very strange! I'll take care of it. 

Is it super urgent for you or something which can wait for a few days?

**Comment by stevebower59** [comment]
Thanks Chris.  The problem occurred when moving the API from our DEV environment to QA.  I believe we'll need to port the same API to PROD either the end of this week, or beginning of the next week, so in the next few days should be fine.

**Comment by cwiechmann** [comment]
Hi @stevebower59, to be clear. This for a Swagger 2.0 API-Specification configured like so:
`"basePath": "/poreceipt"`
having a path:
```
"paths": {
	"/poreceipts": {
```

Can you confirm? I would like to define a Unit-Test for this use-case to replicate the issue.

**Comment by stevebower59** [comment]
This is for a Swagger 1.1 api.

**Comment by stevebower59** [comment]
Configured like
“basePath” : “https://hostname”,
“resourcePath” : “/poreceipt”

...

**Comment by stevebower59** [comment]
paths section is as you have mentioned

“path” : “/poreceipts”

**Comment by cwiechmann** [comment]
@stevebower59, sorry for the late response. Today I wanted to implement a unit-test and realized, that Swagger-Promote is working as expected. With that, I mean the Swagger-File as it is modified by Swagger-Promote is correct.

Therefore I checked, how the API-Manager behaves when importing the API like the following manually using the UI into the API-Manager:
![image](https://user-images.githubusercontent.com/16292002/83485204-d7c8aa00-a4a6-11ea-9ace-f5fdf59a2bff.png)
Unfortunately the API-Manager is misbehaving:
![image](https://user-images.githubusercontent.com/16292002/83485113-9fc16700-a4a6-11ea-9491-f8e1468ce84d.png)

And this is the method:
![image](https://user-images.githubusercontent.com/16292002/83485144-b2d43700-a4a6-11ea-86c5-536d5d6a6f74.png)

I tested with version 7.6.2 and 7.7 2020-March-Release. I recommend to create an Axway support case describing the issue.
